package.json

```diff
@@ -22,9 +22,7 @@
     "csv-parser": "^3.0.0",
     "dompurify": "^2.4.1",
```

Comment:
Shall we remove `dompurify`? It seems only be used in `server/olly/utils/output_builders/build_outputs.ts`. This file already be deleted.